src/release/breaking-changes/page-transition-replaced-by-ZoomPageTransitionBuilder.md

```diff
@@ -0,0 +1,145 @@
+are all switched to the `ZoomPageTransitionsBuilder`.
+When the current `TargetPlatform` doesn't have
+`PageTransitionsBuilder` defined in the `ThemeData.pageTransitionsTheme`,
+`ZoomPageTransitionsBuilder` will be use as default.
+
+[Style guide for Flutter repo]: {{site.repo.flutter}}/wiki/Style-guide-for-Flutter-repo
+
+## Description of change
+
+`PageTransitionsBuilder`s defined in
+`PageTransitionsTheme._defaultBuilders` have changed from
+`FadeUpwardsPageTransitionsBuilder` to
+`ZoomPageTransitionsBuilder` for `TargetPlatform.android`,
+`TargetPlatform.linux` and `TargetPlatform.windows`.
+
+## Migration guide
+
+If you want to switch back to the previous page transition builder
+(`FadeUpwardsPageTransitionsBuilder`), you should define builders
+explicitly for the target platforms.
+
+Code before migration:
+
+<!-- skip -->
+```dart
+MaterialApp(
+  theme: ThemeData(primarySwatch: Colors.blue),
+)
+```
+
+Code after migration:
+
+<!-- skip -->
+```dart
+MaterialApp(
+  theme: ThemeData(
+    pageTransitionsTheme: const PageTransitionsTheme(
+      builders: <TargetPlatform, PageTransitionsBuilder>{
+        TargetPlatform.android: FadeUpwardsPageTransitionsBuilder(), // Apply this to every platforms you need.
+      },
+    ),
+  ),
+)
+```
+
+If you want to apply the same page transition builder to all platforms:
+
+<!-- skip -->
+```dart
+MaterialApp(
+  theme: ThemeData(
+    pageTransitionsTheme: PageTransitionsTheme(
+      builders: Map<TargetPlatform, PageTransitionsBuilder>.fromIterable(
+        TargetPlatform.values,
+        value: (dynamic _) => const FadeUpwardsPageTransitionsBuilder(),
+      ),
+    ),
+  ),
+)
+
+```
+
+### Tests migration
+
+If you used to find widgets but failed with *Too many elements*
+using the new transition, and errors are like these:
+
+```
+══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
+The following StateError was thrown running a test:
+Bad state: Too many elements
+
+When the exception was thrown, this was the stack:
+#0      Iterable.single (dart:core/iterable.dart:656:24)
+#1      WidgetController.widget (package:flutter_test/src/controller.dart:69:30)
+#2      main.<anonymous closure> (file:///path/to/your/test.dart:1:2)
+```
+
+You should migrate your tests by using the
+`descendant` scope for `Finder`s with the specific widget type.
+Below is the example of `DataTable`'s test:
+
+Test before migrate:
+```dart
+final Finder finder = find.widgetWithIcon(Transform, Icons.arrow_upward);
+```
+
+Test after migrate:
+```dart
+final Finder finder = find.descendant(
+  of: find.byType(DataTable),
+  matching: find.widgetWithIcon(Transform, Icons.arrow_upward),
+);
+```
+
+Widgets that typically needs to migrate the finder scope are:
+`Transform`, `FadeTransition`, `ScaleTransition`, and `ColoredBox`.
+
+## Timeline
+
+Landed in version: v2.9.0-1.0.pre<br>
+In stable release: Not yet
```

Comment:
I'm guessing this documentation should be held until this is in stable.

Reply:
The PR should be held until the change is rolled. But shouldn't a breaking change note released along with the change? The *In stable release* should update after.

Reply:
Ahh, yes you are right.

Reply:
Yes, @AlexV525 is correct.  :)